### PR TITLE
Adds checks into pull request workflow

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

**Comment by AlexanderYastrebov** [comment]
Extra checks add 68s to test time as [seen here](https://github.com/zalando/skipper/pull/1483/checks?check_run_id=857620360)

File: filters/auth/oidc.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ type (
 
 	tokenContainer struct {
 		OAuth2Token *oauth2.Token          `json:"oauth2token"`
-		OIDCIDToken string            	   `json:"oidctoken"`
+		OIDCIDToken string                 `json:"oidctoken"`
```
Comment (szuecs) [inline filters/auth/oidc.go:80]: can you drop this one from the PR please :)

**Comment by szuecs** [comment]
👍

**Comment by muaazsaleem** [comment]
👍

**Pull request merged**